ddl.dic

```diff
@@ -715,6 +715,125 @@ save_dictionary_audit.version
 
 save_
 
+save_DICTIONARY_AUTHOR
+
+    _definition.id                DICTIONARY_AUTHOR
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-06-02
+    _description.text
+;
+    The CATEGORY of data items used for dictionary author(s) details.
+;
+    _name.category_id             DICTIONARY
+    _name.object_id               DICTIONARY_AUTHOR
+    _category_key.name            '_dictionary_author.id'
+
+save_
+
+save_dictionary_author.address
+
+    _definition.id                '_audit_author.address'
+    _definition.update            2023-06-02
+    _description.text
+;
+    The address of an author of this dictionary. If there are
+    multiple authors, _dictionary_author.address is looped with
+    _dictionary_author.name.
+;
+    _name.category_id             dictionary_author
+    _name.object_id               address
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+    _description_example.case
+;
+    Department
+    Institute
+    Street
+    City and postcode
+    COUNTRY
+;
+
+save_
+
+save_dictionary_author.id
+
+    _definition.id                '_dictionary_author.id'
+    _definition.update            2023-06-02
+    _description.text
+;
+    Arbitrary identifier for this author.
+;
+    _name.category_id             dictionary_author
+    _name.object_id               id
+    _type.purpose                 Key
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Word
+    _method.purpose               Definition
+    _method.expression
+;
+    _enumeration.default = Unique_id(dictionary_author.id)
+;
```

Comment:
This dREL is not needed since the id item belongs to a newly added category.

Reply:
So because its a new category, an ID is automatically generated if it isn't there?

Reply:
Not exactly. See the following comment (https://github.com/COMCIFS/cif_core/pull/406#discussion_r1215616453) on a similar topic.

In short, the key should be automatically generated for all categories if certain conditions are met (e.g. there is a single loop entry, the key has a arbitrary value and is not referenced by other items).

Note, that the current version of the `ddlm_validate` software used in the dictionary check actions does not yet properly handle such cases, but this is the drawback of the software and not the definitions.

Reply:
> In short, the key should be automatically generated for all categories if certain conditions are met (e.g. there is a single loop entry, the key has a arbitrary value and is not referenced by other items).

I did not know that. That is good behaviour.